scene/gui/color_picker.cpp

```diff
@@ -932,42 +963,18 @@ void ColorPicker::_w_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
-void ColorPicker::_preset_input(const Ref<InputEvent> &p_event) {
+void ColorPicker::_preset_input(const Ref<InputEvent> &p_event, Color p_color) {
```

Comment:
Change it to `const Color& p_color`

Reply:
Fixed. Changed the color parameter to be passed as a const reference also for the add_preset_button(...) function.